framework/auth/oauth_scopes.py

```diff
@@ -41,6 +41,10 @@ class CoreScopes(object):
     APPLICATIONS_READ = 'applications_read'
     APPLICATIONS_WRITE = 'applications_write'
 
+    NODE_LOG_READ = 'nodes.logs_read'
+
+    NULL = 'null'
```

Comment:
I'm not sure I follow the distinction you make.

At any rate, we'll never let users to CUD operations on NodeLogs.